views/js/runner/plugins/security/fullScreen.js

```diff
@@ -236,6 +236,10 @@ define([
             const config = this.getConfig();
             let waitingForUser = false;
 
+            const debouncedFocusListener = _.debounce(() => {
+                handleFullScreenChange();
+            }, 50, { leading: false, trailing: true });
```

Comment:
**quibble:** As per l[odash documentation](https://lodash.com/docs/4.17.15#debounce), it seems `{ leading: false, trailing: true }` is already the default config so this is a bit redundant. Anyway, this is not a big issue.